bio/strling/extract/meta.yaml

```
@@ -0,0 +1,4 @@
+name: "strling extract"
+description: STRling (pronounced like “sterling”) is a method to detect large STR expansions from short-read sequencing data. "Extract" retrieves informative read pairs to a binary format for a single sample (same as above, you can use the same bin files). (https://strling.readthedocs.io/en/latest/run.html)
```

I'd skip the `like` and put every sentence on a separate line, for cleaner version control.
```suggestion
description: >
STRling (pronounced “sterling”) is a method to detect large STR expansions from short-read sequencing data.
``extract`` retrieves informative read pairs to a binary format for a single sample (same as above, you can use the same bin files).
Documentation at: https://strling.readthedocs.io/en/latest/run.html
```